problems/strings.js

```diff
@@ -77,7 +111,14 @@ function verbing() {}
  * @returns {string}
  */
 
-function removeLastWord() {}
+function removeLastWord(str) {
+  if (str === "") {
+    return str;
+  }
+  str.slice(0, -1)
+  str.split(" ")
```

Comment:
slice and split don't mutate. So if you want to have their results you'll need to save them to a variable. Think about splitting earlier and maybe using common array  method to take of the last element.